app/src/main/res/xml/settings.xml

```diff
@@ -75,11 +75,10 @@
     <PreferenceCategory>
 
         <Preference
-            android:icon="@drawable/ic_update"
+            android:icon="@drawable/ic_download"
```

Comment:
Why?

Reply:
it is so to make the drawable differ from manual update check button, no drawable has been added, it is existing.